package.json

```diff
@@ -1,44 +1,44 @@
 {
-  "name": "app-backend",
-  "version": "0.1.0",
-  "description": "The backend API for the Coding Garden community app.",
-  "main": "src/index.js",
-  "scripts": {
-    "start": "cross-env NODE_ENV=production node src/index.js",
-    "dev": "cross-env NODE_ENV=development nodemon src/index.js",
-    "test": "NODE_ENV=test jest --verbose --forceExit",
-    "lint": "eslint src/**/*.js",
-    "seed": "node src/tasks/seed.js",
-    "deploy": "./deploy.sh"
-  },
-  "repository": "https://github.com/CodingGardenCommunity/app-backend.git",
-  "author": "Coding Garden Community App",
-  "license": "MIT",
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn run lint && yarn run test"
+    "name": "app-backend",
+    "version": "0.1.0",
+    "description": "The backend API for the Coding Garden community app.",
+    "main": "src/index.js",
+    "scripts": {
+        "start": "cross-env NODE_ENV=production node src/index.js",
+        "dev": "cross-env NODE_ENV=development nodemon src/index.js",
+        "test": "cross-env NODE_ENV=test jest --verbose --forceExit",
```

Comment:
Since the database connection ins't closed, I get this error after running the tests.
`Have you considered using "--detectOpenHandles" to detect async operations that kept running after all tests finished?`

Should we add "detectOpenHandles" to the test, close database connection after tests have been run, or just ignore the warning message?

Reply:
I added it, I think we are fine with using detectOpenHandles.